ansible/roles/nginx/tasks/main.yml

```diff
@@ -1,6 +1,12 @@
 ---
+  - name: Remove any existing NGINX under "/usr/local" when nginx_auth_enable is true
+    when: nginx_auth_enable is defined and nginx_auth_enable|bool
```

Comment:
Shouldn't this be removed either way (i.e., whether or not `nginx_auth_enable` is true)?